custom_components/vivint/switch.py

```diff
@@ -62,6 +62,18 @@ async def async_setup_entry(
                             device=device, hub=hub, entity_description=PRIVACY_MODE
                         )
                     )
+                if has_capability(device, Category.CAMERA, Capability.DETER):
+                    entities.append(
+                        VivintSwitchEntity(
+                            device=device, hub=hub, entity_description=DETER_MODE
+                        )
+                    )
+                if has_capability(device, Category.DOORBELL, Capability.CAN_CHIME):
+                    entities.append(
+                        VivintSwitchEntity(
+                            device=device, hub=hub, entity_description=DETER_MODE
+                        )
+                    )
```

Comment:
This seems like the wrong capability to check. I don't imagine there is a doorbell that can't chime so this would add it to every doorbell.

Reply:
Yeah oddly enough the Doorbell camera doesn't have the `DETER` capability at least mine doesn't. I'll update the PR to check for another `Capability`

Reply:
We can instead check the Camera for features. One of the features that returns on Camera is `deter`